<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="fullname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="subj_id">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="people_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="cal_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>272</width>
      <height>149</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="schedule_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Checkin_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>540</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckIn</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="checkin_what_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="schedule_what_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="contact_table">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="visit_table">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>401</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="record_contact_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>740</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_contact_button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>740</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visit_info_button">
    <property name="geometry">
     <rect>
      <x>819</x>
      <y>420</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Visit Info</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_note_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>% is wildcard</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_id_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="visit_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Visits</string>
    </property>
   </widget>
   <widget class="QLabel" name="visit_label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>450</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contacts</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
